src/main/scala/org/broadinstitute/dsde/workbench/sam/api/AdminResourceRoutes.scala

```diff
@@ -96,6 +95,15 @@ trait AdminResourceRoutes extends UserInfoDirectives with SecurityDirectives wit
       }
     }
 
+  private def getAdminResourcePolicies(resource: FullyQualifiedResourceId, userInfo: UserInfo, samRequestContext: SamRequestContext): server.Route =
+    get {
+      requireAction(resource, SamResourceActions.adminReadPolicies, userInfo.userId, samRequestContext) {
```

Comment:
We'll actually want to check the `resource_type_admin` resource here instead of the resource we're listing policies for. The user may not have any permissions directly on the resource we're listing policies for, but if they're an admin then they'll have this action on the `resource_type_admin` resource

```suggestion
      requireAction(FullyQualifiedResourceId(SamResourceTypes.resourceTypeAdminName, ResourceId(policyId.resource.resourceTypeName.value)), SamResourceActions.adminReadPolicies, userInfo.userId, samRequestContext) {
```